src/serde_value_ext.rs

```diff
@@ -71,7 +71,7 @@ mod tests {
 
     #[test]
     fn test_render_value_nested_str() {
-        let value = json!({"a": "{{foo.bar.baz}}"});
```

Comment:
These test should have failed when we migrated to jq syntax, do u know why it did not fail?